enterprise/frontend/src/metabase-enterprise/clean_up/CollectionCleanupAlert/CollectionCleanupAlert.tsx

```diff
@@ -0,0 +1,75 @@
+import { c } from "ttag";
+
+import { skipToken } from "metabase/api";
+import Link from "metabase/core/components/Link";
+import { color } from "metabase/lib/colors";
+import { useSelector } from "metabase/lib/redux";
+import * as Urls from "metabase/lib/urls";
+import { getUserIsAdmin } from "metabase/selectors/user";
+import { Alert, Box, Icon } from "metabase/ui";
+import { useListStaleCollectionItemsQuery } from "metabase-enterprise/api/collection";
+import type { Collection } from "metabase-types/api";
+
+const TEXT = {
+  CLEAN_THINGS_UP: c(
+    "This is the heading of a banner that invites the user to clean up a collection.",
+  ).t`Clean things up!`,
+  GET_RID_OF_UNUSED_CONTENT: c(
+    "This is the heading of a banner that invites the user to clean up a collection.",
+  ).t`Get rid of unused content`,
```

Comment:
I was talking to Vitorio yesterday and there is some ambiguity on the scope of this process. I think we could add something like

`Get rid of unused content in this collection` to clarify tht this applies only to the selected collection (and potentially sub-collections)